tests/integration/frameworks/eleventy.test.mjs

```diff
@@ -0,0 +1,158 @@
+import { Buffer } from 'buffer'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+import fetch from 'node-fetch'
+import { afterAll, beforeAll, describe, test } from 'vitest'
+
+import { clientIP, originalIP } from '../../lib/local-ip.cjs'
+import { startDevServer } from '../utils/dev-server.cjs'
+
+// eslint-disable-next-line no-underscore-dangle
+const __dirname = path.dirname(fileURLToPath(import.meta.url))
+
+const context = {}
+
+beforeAll(async () => {
+  const server = await startDevServer({ cwd: path.join(__dirname, '__fixtures__/eleventy-site') })
+
+  context.server = server
+})
+
+afterAll(async () => {
+  const { server } = context
+  await server.close()
+})
+
+describe('eleventy', () => {
+  test('homepage', async (t) => {
+    const { url } = context.server
+    const response = await fetch(`${url}/`).then((res) => res.text())
+
+    t.expect(response.includes('Eleventy Site')).toBe(true)
+  })
+
+  test('redirect test', async (t) => {
+    const { url } = context.server
+    const response = await fetch(`${url}/something`, {
+      redirect: 'manual',
+    })
+    const { headers, status } = response
+
+    t.expect(status).toBe(301)
+    t.expect(headers.get('location').endsWith('/otherthing')).toBe(true)
+    t.expect(await response.text()).toEqual('Redirecting to /otherthing')
+  })
+
+  // TODO: un-skip this once https://github.com/netlify/cli/issues/1242 is fixed
```

Comment:
It looks like this issue has been closed - does this test pass if not skipped?

Reply:
Yes, it pass.
I'll remove the skip and comment.